src/main/java/net/comments/objects/BCommentActions.java

```diff
@@ -0,0 +1,28 @@
+package net.comments.objects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class BCommentActions implements CommentActions {
+    private final WebDriver driver;
+    private CommentTable selectedItem;
+
+    public BCommentActions(WebDriver driver) {
+        this.driver = driver;
+        selectedItem = new BCommentTable(driver);
+    }
+
+    public void duplicate(String comments) {
+        throw new UnsupportedOperationException("Please implement");
+    }
+
+    public void edit(String comments) {
+        throw new UnsupportedOperationException("Please implement");
+    }
+
+    public void delete(int commentsNumber) {
+        selectedItem.selectComment(commentsNumber);
```

Comment:
This line has to be not a part of this method. What if you have to select none or several comments? Select comments int the test directly.